### ISSUE TITLE
env.get type is string instead of any

### ISSUE THREAD

**Issue by gitmacer** [node-red/node-red#4137]
### Current Behavior

Error message if you calculate or comapare with for example ">" and env.get.

### Expected Behavior

Like it were in versions before return type any instead of string so you can calculate and compare without error message.

### Steps To Reproduce

![env get comparing](https://user-images.githubusercontent.com/37345589/232227558-45c62a16-2e3a-452f-a486-96554e9106e9.gif)

### Example flow

```
[
    {
        "id": "06c76eaa0712fe52",
        "type": "function",
        "z": "69d5338b7a2e661d",
        "name": "function 2",
        "func": "if (env.get(\"test\") > 10){\n    \n}\nreturn msg;",
        "outputs": 1,
        "noerr": 1,
        "initialize": "",
        "finalize": "",
        "libs": [],
        "x": 1020,
        "y": 120,
        "wires": [
            []
        ]
    }
]
```

### Environment

- Node-RED version: v3.0.2
- Node.js version: v16.19.1
- npm version:
- Platform/OS: Win 11 64bit
- Browser: firefox 112.0 (64-Bit)

**Comment by knolleary** [comment]
Environment variables are evaluated as strings - that has always been the case. I'm not sure when the typing got changed on this (@Steve-Mcl maybe knows or I have forgotten).

JavaScript does let you compare string representations of numbers and regular numbers, but it isn't always type-safe to do so.

I can see why the type info was changed to be more precise - to make sure you know you are comparing a string with another type.

**Comment by gitmacer** [comment]
If I understand you correctly it should always return a string when getting the env var but I tested it and it is still returning a number if I set the env var to a number.

![grafik](https://user-images.githubusercontent.com/37345589/232231982-796f232e-5f38-4970-9649-397efe1c6142.png)
![grafik](https://user-images.githubusercontent.com/37345589/232231937-5102d9d1-610d-4a56-96cd-b1fd01bda4a3.png)

**Comment by knolleary** [comment]
Ah yes.

Env vars defined *within* Node-RED can be typed. Env vars defined as normal process level env vars are always strings.

So, I think you're right - the typing of the function *should* be `any`

**Comment by Steve-Mcl** [comment]
Agreed. Only process env vars had been considered (which are [always] strings). As `env.get` also hndles the Node-RED env properties, it should be changed

From:
```ts
static get(name:string) :string;
```

To:
```ts
static get(name:string) :any;
```


Here: https://github.com/node-red/node-red/blob/53f99ecc23bae1e99c4a8f87ebd1c9ac8462d1fa/packages/node_modules/@node-red/editor-client/src/types/node-red/func.d.ts#L284

**Comment by knolleary** [comment]
We've updated the typings in our upstream monaco project, but now need to get those changes filtered into this repo.

**Comment by Steve-Mcl** [comment]
fixed in https://github.com/node-red/node-red/pull/4189
